index.html

```diff
@@ -5,222 +5,219 @@
-                        <li><button class="html-c" type="button">HTML</button></li>
-                        <li><button class="bootstrap-c" type="button">Bootstrap</button></li>
-                        <li><button class="ruby-c" type="button">Ruby</button></li>
-                    </ul>
-                    <button class="s-button" type="button">See project</button>
-                </article>
-                <article class="art-data">
-                    <img class="bg-img2" src="assets/recent_works/bg-img.png" alt="background-image"/>
-                    <div class="overlay-content">
-                        <h3 class="title-post">Professional Art <br> Printing Data</h3>
-                        <p class="description">
-                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
-                        </p>
-                        <ul class="stack-2">
-                            <li><button class="html" type="button">HTML</button></li>
-                            <li><button class="bootstrap" type="button">Bootstrap</button></li>
-                            <li><button class="ruby" type="button">Ruby</button></li>
-                        </ul>
+                    <div class="recent-works">
+                        <h2 class="title">My Recent Works</h2>
+                        <hr class="custom-hr">
                     </div>
-                    <div class="grid-container">
-                        <div class="grid-item">
-                            <button class="b-button" type="button">See project</button>
+                    <div class="recent-projects">
+                        <img class="custom-img-mobile" src="assets/recent_works/hm_img.png" alt="placeholder img"/>
+                        <img class="custom-img-desktop" src="assets/recent_works/bg-img-desktop.png" alt="placeholder img"/>
+                        <div class="stories">
+                            <h3 class="s-title">Multi-Post Stories</h3>
+                            <p class="paragraph">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <ul class="stack">
+                                <li><button class="css-c" type="button">CSS</button></li>
+                                <li><button class="html-c" type="button">HTML</button></li>
+                                <li><button class="bootstrap-c" type="button">Bootstrap</button></li>
+                                <li><button class="ruby-c" type="button">Ruby</button></li>
+                            </ul>
+                            <button class="s-button" type="button">See project</button>
```

Comment:
👍 The card elements look good, but they do require some adjustments.

- [ ] First, some elements don't have the proper colors and font styles. I recommend carefully checking each element and updating your code to match the design's specified colors and font styles.

- [ ] Second, the gap among these elements should be consistent as per the design. To achieve this, follow the design closely and adjust the spacing between elements accordingly.

![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/55840999/eab878a4-dbe5-400b-b450-e3b90cd81caf)
By making these adjustments, your card elements will be visually aligned with the design, resulting in a more polished and cohesive appearance. Keep up the great work, and I'm confident your attention to detail will elevate the overall quality of your project! 🎨📝🚀